src/constants.jl

```diff
@@ -598,6 +625,11 @@ const lnm5 = [2n-1 for n in 1:6]                         # (2n - 1)
 const lnm6 = [-(n+1) for n in 1:6]                       # -(n + 1)
 const lnm7 = [m for m in 1:6]                            # m  
 
+# Number of bodies in extended-body accelerations
+const N_ext = 11     
+# Number of bodies in backward integration
+const N_bwd = 11                   
```

Comment:
Backward integration is used to evaluate the ephemeris with a time delay in order to compute tidal interactions . I understand that such interactions do not consider all the bodies. However, @PerezHz may have a better explanation.

Reply:
I see... Then, maybe, it is worth adding simply that `N_bwd` is used for the tidal integrations, and not for the (time) backward integration, which was the origin of my confussion.

Reply:
Indeed, in the comment "backward integration" refers to the short integrations that are done in order to compute the tidal interactions which involve a delay in time; I agree it can be confusing and adding a clarification can help a lot

Reply:
I've changed the comment to
```julia
# Number of bodies used to compute time-delayed tidal interactions
```